plugins/woocommerce/src/Admin/API/OnboardingProfile.php

```diff
@@ -309,6 +332,44 @@ public function get_profile_progress( $request ) {
 		return rest_ensure_response( $onboarding_progress );
 	}
 
+
+	/**
+	 * Update store's currency and measurement units.
+	 *
+	 * @param  WP_REST_Request $request Request data.
+	 * @return WP_Error|WP_REST_Response
+	 */
+	public function update_store_currency_and_measurement_units( WP_REST_Request $request ) {
```

Comment:
Should we have unit tests for this?

Reply:
@rjchow Good idea! Added in [Add tests](https://github.com/woocommerce/woocommerce/pull/55463/commits/def8f1454db04e77242f8d30cec35797541d6e8a)